src/Components/CreateImageWizardV2/validators.ts

```diff
@@ -36,3 +38,24 @@ export const isBlueprintNameValid = (blueprintName: string) =>
 export const isBlueprintDescriptionValid = (blueprintDescription: string) => {
   return blueprintDescription.length <= 250;
 };
+export const isFileSystemConfigValid = (partition: Partition[]) => {
+  const duplicates = getDuplicateMountPoints(partition);
+  return !(duplicates?.length !== 0);
+};
+export const getDuplicateMountPoints = (partitions: Partition[]) => {
+  if (!partitions) {
+    return [];
+  }
+  const mountPointFreqs = {} as { [key: string]: number | boolean };
```

Comment:
I think this is a bit confusing, especially how the value can be either a number or boolean.

What if we used a `Set()` to keep track of the duplicates? I'm not sure if we should return the set itself or convert it to an array... we don't use sets that often in our code base so returning an array might make it a bit easier to work with, because we will be able to call array methods on it...

Reply:
I think if we take this approach we can probably also get rid of that early return...